lib/main.coffee

```diff
@@ -351,10 +351,10 @@ module.exports = Hydrogen =
             @wsKernelPicker = new WSKernelPicker (kernel) =>
                 @clearResultBubbles()
 
-                grammar = @editor.getGrammar()
+                grammar = kernel.grammar
```

Comment:
The problem is that `@kernelManager.destroyRunningKernelFor` isn't working with `kernel.grammar` because kernels are identified via `@editor.grammar` (e.g. `python` instead of `python 3`).

I think we should use `@editor.getGrammar()` for now.